README.md

```diff
@@ -93,10 +107,19 @@ Now in a terminal window, run:
 ```
 cd /usr/local/lib/
 mkdir share && cd share
-mkdir oracle && cd oracle
+mkdir oracle && cd oracle 
 tar -xzf ~/Downloads/instantclient-basic-macos.x64-12.1.0.2.0.zip
 tar -xzf ~/Downloads/instantclient-sdk-macos.x64-12.1.0.2.0.zip
```

Comment:
Correct me if I'm wrong I believe we were forced to use a newer basic/sdk? 19.*? I believe the earlier ones are missing some files.

Reply:
I was sure but I think I'll change it anyways